newswires/client/src/Item.tsx

```diff
@@ -18,6 +19,7 @@ export const Item = ({ id }: { id: string }) => {
 
 	const [itemData, setItemData] = useState<WireData | undefined>(undefined);
 	const [error, setError] = useState<string | undefined>(undefined);
+	const [showJson, setShowJson] = useState<boolean>(false);
```

Comment:
pedantic, and you might disagree, but I find `should` or `is` prefixes useful for boolean variables (when I see `showJson` I assume that's a function...
```suggestion
	const [shouldShowJson, setShouldShowJson] = useState<boolean>(false);
```